### PR TITLE
:hammer: fix for ssh failures using key files

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:
Fix ssh issue with key files.
- With k8s 1.24 version , ssh using key files is failing. Updating the crypto package to latest version fixes it.

**Which issue(s) this PR fixes** (optional)
Closes #PTX-12713

**Special notes for your reviewer**:

Result: https://jenkins.pwx.dev.purestorage.com/job/Torpedo/view/Torpedo%20NextPX%20Cloud/job/tp-nextpx-asggke/455/console

https://jenkins.pwx.dev.purestorage.com/job/Torpedo/job/tp-nextpx-asggke/463/console

golint output:

```
~/workspace/px-test-repo/go/src/github.com/portworx/torpedo   topic/leela/gke-ptx-12713→origin/master ↑2                                                                                                                                                                                      ✔  10241  12:49:18
 $ for file in $(find . -name '*.go' | grep -v vendor | grep -v '\.pb\.go' | grep -v '\.pb\.gw\.go'); do \

        golint ${file}; \

        if [ -n "$(golint ${file})" ]; then \

                exit 1; \

        fi; \

done
 ~/workspace/px-test-repo/go/src/github.com/portworx/torpedo   topic/leela/gke-ptx-12713→origin/master ↑2                                                                                                                                                                                      ✔  10242  12:49:28
 $ 


```

### CODE REVIEW

**Review by jainsmit — COMMENTED** [review]
Please fix the build errors. As multiple files got changed, can you please run some regression for safe side ..

**Comment by lsrinivas-pure** [comment]
> Details

Hi @jainsmit 

There seems to issue with Travis thats why checks are failing. I have run local compile and lint and it is working fine.
Im working Devops team for Travis issue and can merge it with their help once I got the approval.

Regression run :  https://jenkins.pwx.dev.purestorage.com/job/Torpedo/view/Torpedo%20NextPX%20Cloud/job/tp-nextpx-asggke/455/console

this includes regression also

**Review by jainsmit — APPROVED** [review]
LGTM !!

**Pull request merged**
